app/views/About.js

```diff
@@ -12,14 +12,14 @@ import {
 } from 'react-native';
 
 import { useStrategyContent } from '../TracingStrategyContext';
-import packageJson from '../../package.json';
 import { NavigationBarWrapper, Typography } from '../components';
 
 import { useDispatch } from 'react-redux';
 import toggleAllowFeatureFlagsAction from '../store/actions/featureFlags/toggleAllowFeatureFlagsEnabledAction';
 import { Colors, Spacing, Typography as TypographyStyles } from '../styles';
 
 const CLICKS_TO_ENABLE_FEATURE_FLAGS = 10;
+const APP_VERSION = 'ALPHA.1.0.0.b7';
```

Comment:
you can use `react-native-device-info` package to get the app version, saving you the trouble of having to manually update this every time.

Reply:
Thanks, the package is not yet added to the codebase, I'll add it now.

Reply:
I also added the build number to the About screen because on iOS the version would always be `1.0.0` until we decide to bump it, making it harder for testers to know which exact version they are using.